src/background/service/APIService.js

```diff
@@ -325,14 +366,17 @@ class APIService {
             }
 
             let privKeyEncrypt = await this.encryptor.encrypt(this.getStore().password, wallet.privKey_hex)
-            const account = {
+            let account = {
```

Comment:
What does this mean, can you explain it?
there change because when wallet have evmAddress , account will add evmAddress
[this link](https://github.com/oasisprotocol/oasis-wallet-ext/blob/b5b69d732a2c0c345a1be2db5fc30ff7beb62bb3/src/background/service/APIService.js#L377)

Reply:
https://jsfiddle.net/afeb9m13/

see this sample snippet. you should be able to set `.evmAddress` on a `const`

Reply:
(I actually like leaving variables with deep mutations as `let` too :)

Reply:
Can you talk about why you want to change to `const`? For variable parameters, I prefer to use `let`

Reply:
it enforces that later changes to the code doesn't accidentally reassign the variable. but your guess is as good as mine whether we'll first encounter that or we'll first want to reassign it for real. feel free to keep it this way, given that you prefer this.

Reply:
gonna resolve this